TeamCode/src/main/java/eaglesfe/common/RoverRuckusRobotPositionEstimator.java

```diff
@@ -67,28 +72,28 @@ public void initialize(HardwareMap hardwareMap, VuforiaLocalizer.CameraDirection
         VuforiaTrackable blueRover = targetsRoverRuckus.get(0);
         blueRover.setName("Blue-Rover");
         OpenGLMatrix blueRoverLocationOnField = OpenGLMatrix
-                .translation(0, VuforiaBase.MM_FTC_FIELD_WIDTH, mmTargetHeight)
+                .translation(0, VuforiaBase.MM_FTC_FIELD_WIDTH / 2, mmTargetHeight)
```

Comment:
This is where the "divide by two error" was. We were making the field 24 feet on a side rather than 12.